src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -375,6 +403,36 @@ const generateId = () => {
         '-' + random4() + random4() + random4();
 };
 
+/*******************************************************************************
+* @description Javascript method comparable to Apex's String.format(...).
+* Replaces placeholders in Custom Labels ({0}, {1}, etc) with provided values.
+*
+* @param {string} string: Custom Label to be formatted.
+* @param {list} replacements: List of string to use as replacements.
+* @return {string} formattedString: Formatted custom label
+*/
+const format = (string, replacements) => {
```

Comment:
Is this a duplicate of the method in geLabelService?

Reply:
Sent a gist with some thoughts around this via quip.